app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,27 @@
+module SessionsHelper
+    
+    # Assign user ID to session ID for login functionality
```

Comment:
It looks like you have 4 space indentation in this file. Can you re-indent it with 2 spaces to match the rest please?